### PR TITLE
fix urls for search

### PR DESCRIPTION
**Pull request by naeluh**

### CODE REVIEW

**Comment by naeluh** [comment]
@geealbers I think this should work for the base url I am going to see about changes to the netlify address - also Dave Lacey is going to fix the redirect tonight.

File: themes/quire-starter-theme/layouts/data/search-index.html [hunk 1]
```diff
@@ -58,7 +58,7 @@
   {{ .Scratch.Set "pageFullTitle" (.Scratch.Get "pageTitle") }}
 {{- end -}}
 
-{{- .Scratch.Set "pageData" (dict "id" $index "title" (.Scratch.Get "pageFullTitle") "url" (.Scratch.Get "pageURL") "type" (.Scratch.Get "pageType") "length" (.Scratch.Get "pageLength") "content" (.Scratch.Get "searchContent")) -}}
+{{- .Scratch.Set "pageData" (dict "id" $index "title" (.Scratch.Get "pageFullTitle") "url" (.Scratch.Get "pageURL" | urlize) "url1" (.Scratch.Get "pageURL" | absURL) "url2" (.Scratch.Get "pageURL" | relURL) "url3" (.Scratch.Get "pageURL" | safeURL) "type" (.Scratch.Get "pageType") "length" (.Scratch.Get "pageLength") "content" (.Scratch.Get "searchContent")) -}}
```
Comment (geealbers) [inline themes/quire-starter-theme/layouts/data/search-index.html]: were`url1`, `url2` and `url3` in there just for testing or are you actually using them all?

File: themes/quire-starter-theme/source/js/search.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ class Search {
   }
 
   buildIndex(data) {
+    console.log(data)
```
Comment (geealbers) [inline themes/quire-starter-theme/source/js/search.js]: do we need to leave this in here?

File: themes/quire-starter-theme/layouts/data/search-index.html (hunk 1 above)
Comment (naeluh) [reply themes/quire-starter-theme/layouts/data/search-index.html]: Oops sorry about that ! I will take them out

File: themes/quire-starter-theme/source/js/search.js (hunk 2 above)
Comment (naeluh) [reply themes/quire-starter-theme/source/js/search.js]: same here ... thanks !

**Comment by naeluh** [comment]
@geealbers ok I updated those files thanks for catching those!

**Comment by geealbers** [comment]
Great. Will merge now and we'll see how it does on getty.edu. Thanks!

**Pull request merged**